pkg/apis/pipeline/v1alpha1/resource_types.go

```diff
@@ -94,7 +94,9 @@ type TaskResource struct {
 	// +optional
 	// TargetPath is the path in workspace directory where the task resource will be copied.
 	TargetPath string `json:"targetPath"`
-	// Resource Value stuff
+	// +optional
+	// Path to the index.json file for output container images
+	OutputImageDir string `json:"outputImageDir"`
```

Comment:
Any reason to not re-use `TargetPath` for the output folder ? (might have been already discussed :bowing_man:)

Reply:
don't know, didn't think about it 🤷🏼‍♂️

Reply:
Asking that as this field is only gonna be useful for one type of resources and I feel it would make sense to re-use `TargetPath`.
@bobcatfish @abayer wdyt ?

Reply:
I think it might bit a little confusing because it will have different meanings

Reply:
> I think it might bit a little confusing because it will have different meanings

Yeah I am 50/50 on that. It's close (as it is a "target" path), but user "may" think it's related to the image reference (although this is not where we define that one so…). I'm not sure how confusing it can be :sweat_smile: #dont-know-what-i-am-talking-about :joy_cat:

Reply:
oooo yeah this is a good idea - I'm really tempted to re-use `targetPath` here too 🤔but also I'm torn because overloading it may be confusing. Since we're alpha (famous last words) I'm thinking go ahead with `OutputImageDir` for now and see how it goes?